packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -298,7 +298,7 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
 
   // When the editor is on full size mode, the user can resize the height of the editor.
   const onMouseDownResizeHandler = useCallback(
-    (mouseDownEvent) => {
+    (mouseDownEvent: any) => {
```

Comment:
I hate anys. LEt's do it correctly. you will need to create a function (add this at the top, under the languageId function)

```
function isMouseEvent(e: React.TouchEvent | React.MouseEvent): e is React.MouseEvent {
  return e && 'pageY' in e;
}

and then change the function as :
```
  const onMouseDownResizeHandler = useCallback(
    (mouseDownEvent: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent) => {
      const startSize = editorHeight;
      const startPosition = isMouseEvent(mouseDownEvent)
        ? mouseDownEvent?.pageY
        : mouseDownEvent?.touches[0].pageY;

      function onMouseMove(mouseMoveEvent: MouseEvent) {
        const height = startSize - startPosition + mouseMoveEvent.pageY;
        const validatedHeight = Math.min(Math.max(height, EDITOR_MIN_HEIGHT), EDITOR_MAX_HEIGHT);
        setEditorHeight(validatedHeight);
      }
      function onMouseUp() {
        document.body.removeEventListener('mousemove', onMouseMove);
      }

      document.body.addEventListener('mousemove', onMouseMove);
      document.body.addEventListener('mouseup', onMouseUp, { once: true });
    },
    [editorHeight]
  );
```
```